components/tools/OmeroWeb/omeroweb/settings.py

```diff
@@ -1248,6 +1269,10 @@ def report_settings(module):
     }
 }
 
+SESSION_COOKIE_HTTPONLY = True
```

Comment:
Can I suggest that these are included as configuration properties but with descriptions that include what you've learned about what breaks? This will break people from re-trying what you've already attempted.

Reply:
I can do that for `CSRF_COOKIE_HTTPONLY default False`. I don't think `SESSION_COOKIE_HTTPONLY` should be configurable though, there's no reason it should be `False`.